packages/types/src/user.ts

```diff
@@ -49,10 +49,7 @@ export interface UserResource extends ClerkResource {
   firstName: string | null;
   lastName: string | null;
   profileImageUrl: string;
```

Comment:
❓ should we mark the `profileImageUrl`  as `@deprecated` ?

Reply:
We will  as part of a different PR as we need to make sure docs are online before the deprecated comment is added